Subject: mkspec for ptxdist
From: Michael Olbrich <m.olbrich@pengutronix.de>

Add a mkspec for ptxdist:
- qplatformdefs.h
    Include the generic qplatformdefs.h
- qmake.conf.in
    Used to generate qmake.conf with the correct toolchain.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 mkspecs/linux-ptx-g++/qmake.conf.in       |   35 ++++++++++++++++++++++++++++++
 mkspecs/linux-ptx-g++/qplatformdefs.h     |    2 +
 mkspecs/qws/linux-ptx-g++/qmake.conf.in   |   25 +++++++++++++++++++++
 mkspecs/qws/linux-ptx-g++/qplatformdefs.h |    2 +
 4 files changed, 64 insertions(+)

Index: b/mkspecs/qws/linux-ptx-g++/qmake.conf.in
===================================================================
--- /dev/null
+++ b/mkspecs/qws/linux-ptx-g++/qmake.conf.in
@@ -0,0 +1,25 @@
+#
+# qmake configuration for linux-ptx-g++
+#
+
+
+include(../../common/g++.conf)
+include(../../common/linux.conf)
+include(../../common/qws.conf)
+
+# modifications to g++.conf
+QMAKE_CC                = @COMPILER_PREFIX@gcc
+QMAKE_CXX               = @COMPILER_PREFIX@g++
+QMAKE_INCDIR            = @INCDIR@
+QMAKE_LIBDIR            = @LIBDIR@
+
+QMAKE_LINK              = @COMPILER_PREFIX@g++
+QMAKE_LINK_SHLIB        = @COMPILER_PREFIX@g++
+QMAKE_LFLAGS            = @LDFLAGS@
+
+# modifications to linux.conf
+QMAKE_AR                = @COMPILER_PREFIX@ar cqs
+QMAKE_OBJCOPY           = @COMPILER_PREFIX@objcopy
+QMAKE_STRIP             = @COMPILER_PREFIX@strip
+
+load(qt_config)
Index: b/mkspecs/qws/linux-ptx-g++/qplatformdefs.h
===================================================================
--- /dev/null
+++ b/mkspecs/qws/linux-ptx-g++/qplatformdefs.h
@@ -0,0 +1,2 @@
+
+#include "../../linux-g++/qplatformdefs.h"
Index: b/mkspecs/linux-ptx-g++/qmake.conf.in
===================================================================
--- /dev/null
+++ b/mkspecs/linux-ptx-g++/qmake.conf.in
@@ -0,0 +1,35 @@
+#
+# qmake configuration for linux-ptx-g++
+#
+
+
+include(../common/g++.conf)
+include(../common/linux.conf)
+
+MAKEFILE_GENERATOR      = UNIX
+TEMPLATE                = app
+CONFIG                 += qt warn_on release incremental link_prl
+QT                     += core gui network
+QMAKE_INCREMENTAL_STYLE = sublib
+
+# modifications to g++.conf
+QMAKE_CC                = @COMPILER_PREFIX@gcc
+QMAKE_CXX               = @COMPILER_PREFIX@g++
+QMAKE_INCDIR            = @INCDIR@
+QMAKE_LIBDIR            = @LIBDIR@
+
+QMAKE_LINK              = @COMPILER_PREFIX@g++
+QMAKE_LINK_SHLIB        = @COMPILER_PREFIX@g++
+QMAKE_LFLAGS            = @LDFLAGS@
+
+QMAKE_INCDIR_X11        =
+QMAKE_LIBDIR_X11        =
+QMAKE_INCDIR_OPENGL     =
+QMAKE_LIBDIR_OPENGL     =
+
+# modifications to linux.conf
+QMAKE_AR                = @COMPILER_PREFIX@ar cqs
+QMAKE_OBJCOPY           = @COMPILER_PREFIX@objcopy
+QMAKE_STRIP             = @COMPILER_PREFIX@strip
+
+load(qt_config)
Index: b/mkspecs/linux-ptx-g++/qplatformdefs.h
===================================================================
--- /dev/null
+++ b/mkspecs/linux-ptx-g++/qplatformdefs.h
@@ -0,0 +1,2 @@
+
+#include "../linux-g++/qplatformdefs.h"
